packages/cli/src/UserManagement/UserManagementHelper.ts

```diff
@@ -61,8 +61,20 @@ export function isValidEmail(email: string): boolean {
 	);
 }
 
-export function isValidPassword(password: string) {
-	return password.length >= 8 && password.length <= 64;
+export function validatePassword(password?: string) {
+	if (!password) {
+		throw new ResponseHelper.ResponseError('Password is mandatory', undefined, 400);
```

Comment:
aren't we building a dependency to `ResponseHelper` here?

Reply:
Same pattern as here: `packages/cli/src/UserManagement/routes/users.ts` in UM and elsewhere in `/cli`.

Should we preferably not use the helper?